tools/create-cluster

```diff
@@ -17,14 +17,6 @@ if [[ ! -z "$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE" ]]; then
     unset OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
 fi
 
-# https://github.com/ironcladlou/openshift4-libvirt-gcp/issues/29
-# gcp image is provisioned with this version, but is auto-updated.
-# rather than turn off auto-upates, downgrade qemu-kvm each go.
-if ! sudo dnf info qemu-kvm | grep -A 5 'Installed Packages' | grep 88.module+el8.1.0+5708+85d8e057.3; then
-    echo "downgrading qemu-kvm to version 2.12.0-88.module+el8.1.0+5708+85d8e057.3"
-    sudo dnf remove -y qemu-kvm && sudo dnf install -y qemu-kvm-2.12.0-88.module+el8.1.0+5708+85d8e057.3
-fi
-
```

Comment:
you've verified this is no longer required?

Reply:
The linked bugzilla indicate that it's fixed with `qemu-kvm-4.2.0-19.module+el8.2.0+6296+6b821950` and newer, and that 8.3 has something newer than this, so it should no longer be required.

Reply:
@sallyom Yes I did verified this one, you can check same for `openshift-gce-devel` project with updated image.

Reply:
nice, ok great